ports/esp32/gccollect.c

```diff
@@ -80,3 +80,15 @@ void gc_collect(void) {
 }
 
 #endif
+
+#if MICROPY_GC_SPLIT_HEAP_AUTO
+
+size_t gc_get_max_new_split(void) {
+    // Need to allow some overhead for heap block metadata.
+    // Am choosing a conservative value as TLSF impl is a bit opaque.
```

Comment:
Suggest `// Choose a conservative value as TLSF implementation is a bit opaque.`